src/nvim/CMakeLists.txt

```diff
@@ -746,74 +746,81 @@ if(WIN32)
       -P ${PROJECT_SOURCE_DIR}/cmake/WindowsDllCopy.cmake)
   add_dependencies(nvim_runtime_deps nvim_dll_deps)
 
+  # option(USE_BUNDLED_NVIMQT "Bundle neovim-qt" OFF)
+  if(USE_BUNDLED_NVIMQT)
```

Comment:
As of now, this conditional always returns false, because it's not defined anywhere.  A variable with the same name is defined in `cmake.deps/CMakeLists.txt` but that folder isn't `add_subdirectory`'d in `src/nvim/CMakeLists.txt`.

I'm leaving this patch here in case someone knows how to solve this problem.

Reply:
You need to define `USE_BUNDLED_NVIMQT` in `CMakeLists.txt` *and* in `cmake.deps/CMakeLists.txt` as they are completely separate cmake files.

Reply:
Hmm, I know.  But that would force the user to set this variable twice: once when they build the deps and once when they build nvim.  I was hoping there is another way.

Reply:
I don't think so.